services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    try:
+        return Movie.objects.get(id=movie_id)
+    except Movie.DoesNotExist:
+        raise "No Movie matches the given query."
```

Comment:
Raising a string is not a valid way to handle exceptions in Python. Consider raising a more appropriate exception, such as `raise ValueError('No Movie matches the given query.')` or using `get_object_or_404` if this is a Django view.